tests/js/client/shell/shell-transaction.js

```diff
@@ -4332,6 +4332,95 @@ function transactionTTLStreamSuite () {
   };
 }
 
+function transactionIteratorSuite() {
+  'use strict';
+  var cn = 'UnitTestsTransaction';
+  var c = null;
+
+  return {
+
+    setUp: function () {
+      db._drop(cn);
+      c = db._create(cn, { numberOfShards: 4 });
+    },
+
+    tearDown: function () {
+
+      if (c !== null) {
+        c.drop();
+      }
+
+      c = null;
+      internal.wait(0);
```

Comment:
I think `c` can not be non-null here:
```suggestion
      db._drop(cn);
```